Source/Scene/Cesium3DTileBatchTable.js

```diff
@@ -0,0 +1,1519 @@
+/*global define*/
+define([
+        '../Core/arrayFill',
+        '../Core/Cartesian2',
+        '../Core/Cartesian4',
+        '../Core/clone',
+        '../Core/Color',
+        '../Core/combine',
+        '../Core/ComponentDatatype',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Core/DeveloperError',
+        '../Core/Math',
+        '../Core/PixelFormat',
+        '../Renderer/ContextLimits',
+        '../Renderer/DrawCommand',
+        '../Renderer/Pass',
+        '../Renderer/PixelDatatype',
+        '../Renderer/RenderState',
+        '../Renderer/Sampler',
+        '../Renderer/ShaderSource',
+        '../Renderer/Texture',
+        '../Renderer/TextureMagnificationFilter',
+        '../Renderer/TextureMinificationFilter',
+        './BlendingState',
+        './Cesium3DTileColorBlendMode',
+        './CullFace',
+        './getBinaryAccessor',
+        './StencilFunction',
+        './StencilOperation'
+    ], function(
+        arrayFill,
+        Cartesian2,
+        Cartesian4,
+        clone,
+        Color,
+        combine,
+        ComponentDatatype,
+        defaultValue,
+        defined,
+        defineProperties,
+        destroyObject,
+        DeveloperError,
+        CesiumMath,
+        PixelFormat,
+        ContextLimits,
+        DrawCommand,
+        Pass,
+        PixelDatatype,
+        RenderState,
+        Sampler,
+        ShaderSource,
+        Texture,
+        TextureMagnificationFilter,
+        TextureMinificationFilter,
+        BlendingState,
+        Cesium3DTileColorBlendMode,
+        CullFace,
+        getBinaryAccessor,
+        StencilFunction,
+        StencilOperation) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    function Cesium3DTileBatchTable(content, featuresLength, batchTableJson, batchTableBinary) {
+        /**
+         * @readonly
+         */
+        this.featuresLength = featuresLength;
+
+        this._translucentFeaturesLength = 0; // Number of features in the tile that are translucent
+
+        /**
+         * @private
+         */
+        this.batchTableJson = batchTableJson;
+
+        /**
+         * @private
+         */
+        this.batchTableBinary = batchTableBinary;
```

Comment:
Only the `pickTexture` and `batchTexture` is accounted for in the stats. CPU mirrors are not accounted for. In reality these batch table buffers are pretty small compared to the geometry and texture memory.

The documentation in `Cesium3DTileset` has since been changed to indicate the memory stats are GPU only.